### ISSUE TITLE
Comment system ?

### ISSUE THREAD

**Issue by valerianrossigneux** [holzenburg/feedshare#3]
Hi, nice work!
Do you plan to add a comment system so that user can interact?

**Comment by holzenburg** [comment]
Yes, I did plan to add features like this. Unfortunately I have no time left to work on this project at the moment. Hopefully this will change soon. In the meantime – you are very welcome to make pull requests!

Am 15.04.2014 um 15:36 schrieb cyberval notifications@github.com:
